src/v/cloud_storage/probe.cc

```diff
@@ -11,90 +11,116 @@
+              sm::description("Number of partition manifest downloads")),
+            sm::make_counter(
+              "manifest_upload_backoff",
+              [this] { return get_manifest_upload_backoffs(); },
+              sm::description(
+                "Number of times backoff was applied during manifest upload")),
+            sm::make_counter(
+              "manifest_download_backoff",
+              [this] { return get_manifest_download_backoffs(); },
+              sm::description("Number of times backoff was applied during "
+                              "manifest download")),
+            sm::make_counter(
+              "successful_uploads",
+              [this] { return get_successful_uploads(); },
+              sm::description("Number of completed log-segment uploads")),
+            sm::make_counter(
+              "successful_downloads",
+              [this] { return get_successful_downloads(); },
+              sm::description("Number of completed log-segment downloads")),
+            sm::make_counter(
+              "failed_uploads",
+              [this] { return get_failed_uploads(); },
+              sm::description("Number of failed log-segment uploads")),
+            sm::make_counter(
+              "failed_downloads",
+              [this] { return get_failed_downloads(); },
+              sm::description("Number of failed log-segment downloads")),
+            sm::make_counter(
+              "failed_manifest_uploads",
+              [this] { return get_failed_manifest_uploads(); },
+              sm::description("Number of failed manifest uploads")),
+            sm::make_counter(
+              "failed_manifest_downloads",
+              [this] { return get_failed_manifest_downloads(); },
+              sm::description("Number of failed manifest downloads")),
+            sm::make_counter(
+              "upload_backoff",
+              [this] { return get_upload_backoffs(); },
+              sm::description("Number of times backoff was applied during "
+                              "log-segment uploads")),
+            sm::make_counter(
+              "download_backoff",
+              [this] { return get_download_backoffs(); },
+              sm::description("Number of times backoff  was applied during "
+                              "log-segment downloads")),
+            sm::make_counter(
+              "bytes_sent",
+              [this] { return _cnt_bytes_sent; },
+              sm::description("Number of bytes sent to cloud storage")),
+            sm::make_counter(
+              "bytes_received",
+              [this] { return _cnt_bytes_received; },
+              sm::description("Number of bytes received from cloud storage")),
+          });
+    }
+
+    if (!public_disabled) {
+        auto direction_label = sm::label("direction");
+
+        _public_metrics.add_group(
+          prometheus_sanitize::metrics_name("shadow_indexing"),
```

Comment:
IMO, "shadow_indexing" makes more sense since that is the feature name; however, I think "cloud_storage" would be more appropriate here since it matches the sub-system name and config prefixes in the yaml config file. We can always change the name when/if "cloud_storage" is changed.

Reply:
Done. Changed the name to cloud_storage.